packages/whale-api-client/__tests__/api/consortium.test.ts

```diff
@@ -188,3 +190,307 @@ describe('getAssetBreakdown', () => {
+    await alice.generate(1)
+    await tGroup.waitForSync()
+  }
+
+  async function setMemberInfo (tokenId: string, memberInfo: Array<{ id: string, name: string, backingId: string, ownerAddress: string, mintLimit: string, mintLimitDaily: string }>): Promise<void> {
+    const infoObjs: { [key: string]: object } = {}
+
+    memberInfo.forEach(mi => {
+      infoObjs[mi.id] = {
+        name: mi.name,
+        ownerAddress: mi.ownerAddress,
+        backingId: mi.backingId,
+        mintLimitDaily: mi.mintLimitDaily,
+        mintLimit: mi.mintLimit
+      }
+    })
+
+    return await setGovAttr({ [`v0/consortium/${tokenId}/members`]: infoObjs })
+  }
+
+  async function setup (): Promise<void> {
+    accountAlice = await alice.generateAddress()
+    accountBob = await bob.generateAddress()
+
+    await alice.token.create({
+      symbol: symbolBTC,
+      name: symbolBTC,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: accountAlice
+    })
+    await alice.generate(1)
+
+    await alice.token.create({
+      symbol: symbolETH,
+      name: symbolETH,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: accountAlice
+    })
+    await alice.generate(1)
+
+    await alice.container.fundAddress(accountBob, 10)
+    await alice.generate(1)
+    idBTC = await alice.token.getTokenId(symbolBTC)
+    idETH = await alice.token.getTokenId(symbolETH)
+
+    await setGovAttr({
+      'v0/params/feature/consortium': 'true',
+      [`v0/consortium/${idBTC}/mint_limit`]: '10',
+      [`v0/consortium/${idBTC}/mint_limit_daily`]: '5',
+      [`v0/consortium/${idETH}/mint_limit`]: '20',
+      [`v0/consortium/${idETH}/mint_limit_daily`]: '10'
+    })
+
+    await setMemberInfo(idBTC, [{
+      id: '01',
+      name: 'alice',
+      ownerAddress: accountAlice,
+      backingId: 'abc',
+      mintLimitDaily: '5',
+      mintLimit: '10'
+    }, {
+      id: '02',
+      name: 'bob',
+      ownerAddress: accountBob,
+      backingId: 'def,hij',
+      mintLimitDaily: '5',
+      mintLimit: '10'
+    }])
+
+    await setMemberInfo(idETH, [{
+      id: '01',
+      name: 'alice',
+      ownerAddress: accountAlice,
+      backingId: '',
+      mintLimitDaily: '10.00000000',
+      mintLimit: '20.00000000'
+    }, {
+      id: '02',
+      name: 'bob',
+      ownerAddress: accountBob,
+      backingId: 'lmn,opq',
+      mintLimitDaily: '10.00000000',
+      mintLimit: '20.00000000'
+    }])
+
+    await alice.rpc.token.mintTokens(`1@${symbolBTC}`)
+    await alice.generate(1)
+
+    await alice.rpc.token.mintTokens(`2@${symbolETH}`)
+    await alice.generate(1)
+
+    await alice.rpc.token.burnTokens(`1@${symbolETH}`, accountAlice)
+    await alice.generate(1)
+    await tGroup.waitForSync()
+
+    await bob.rpc.token.mintTokens(`4@${symbolBTC}`)
+    await bob.generate(1)
+
+    await bob.rpc.token.burnTokens(`2@${symbolBTC}`, accountBob)
+    await bob.generate(1)
```

Comment:
What if both Alice and Bob mint and burn multiple tokens within a single block?

Reply:
Added on [`c37c4b8`](https://github.com/JellyfishSDK/jellyfish/pull/1868/commits/c37c4b80bd1ac5c235bc8bfa16b8bfed7e7e88f6)